README.md

```diff
@@ -11,8 +11,13 @@ Both Social Team members and Node.js project collaborators submitting PRs should
   - Hashtags like #nodejs or #javascript are relatively unused outside of spam at this point, and add no additional semantic value on Twitter.
 - We do not amplify or share content that could unduly promote or benefit one corporation that does work in the space of Node.js over another.
 - We try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
-  - For general releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
-  - For security releases, it is important that the tweet come directly from the Node.js Twitter account (the authoritative source) so that readers will immediately recognize the announcement. Additionally, if there is an announcement tweet from collaborators who put in the effort to make the security release happen, those tweets should also be amplified with a retweet from the Node.js account.
+  - For minor and patch releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
+  - For major releases, it is important that the tweet come directly from the Node.js Twitter account (the authoritative source) for several reasons:
+    - Direct tweets enable readers immediately recognize the announcement from an authoratative source that they know.
+    - Direct tweets for majors helps ensure maximal engagement and reach of the project's release.
+    - Direct tweets ensure that the official message is online and endures, regardless of external factors.
+  - For security releases, it is important that the tweet come directly from the Node.js Twitter account (the authoritative source) so that readers will immediately recognize the announcement. Direct tweets for security releases helps ensure maximal engagement and reach of the project's release.
```

Comment:
I also rewrote this section to be consistent with the major and patch releases section. However, it's technically outside the intended scope of this PR so I'll leave it here as a suggestion and if others think this rewrite should be included here (rather than another PR), I'm happy to commit it.

```suggestion
  - For security releases, it is important that the tweet come directly from the Node.js Twitter account (the authoritative source) for several reasons:
    - Direct tweets enable readers immediately recognize the announcement from an authoratative source that they know.
    - Direct tweets for majors helps ensure maximal engagement and reach of the project's release.
    - Direct tweets ensure that the official message is online and endures, regardless of external factors.  - Additionally, **if**  there is an announcement tweet from Releasers who put in the effort to make any given major release or security release happen, those tweets should also be amplified with a retweet from the Node.js account.
```